packages/forms/src/UIForm/UIForm.component.test.js

```diff
@@ -332,18 +350,39 @@ describe('UIForm component', () => {
 
 		it('should call submit callback when form is valid', () => {
 			// given
-			const validProperties = {
-				...data.properties,
-				lastname: 'This has at least 10 characters',
-				firstname: 'This is required',
-			};
 			const wrapper = mount(<UIFormComponent {...data} {...props} properties={validProperties} />);
 
 			// when
 			wrapper.instance().onSubmit(submitEvent);
 
 			// then
 			expect(props.onSubmit).toBeCalled();
+			expect(props.setErrors).toHaveBeenCalledWith(submitEvent, {}, noop);
+		});
+	});
+
+	describe('#focusFirstError', () => {
+		it('should focus the first element', () => {
+			const focus = jest.fn();
+			const querySelector = jest.fn(() => ({
+				focus,
+			}));
+			const wrapper = shallow(<UIFormComponent {...data} {...props} />);
+			wrapper.instance().formRef = { querySelector };
+			wrapper.instance().focusFirstError();
```

Comment:
Instead of getting an internal function, can you
1- pass a mock as setErrors
2- mount it
3- submit the form with errors
4- get the 3rd argument of setErrors
5- call it
6- check the activeElement

Reply:
done